index.html

```diff
@@ -5,6 +5,9 @@
     <title>Jump Game</title>
     <link rel="stylesheet" href="style.css">
 </head>
+<h1>Welcome to the dinosaur game ripoff</h1>
```

Comment:
How about making it all cap, and get rid of ripoff.

Reply:
first half is good second half is a no